incubator/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/network/disconnect.invalid.session.expiry/server.rpt

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2021-2023 Aklivity Inc.
+#
+# Aklivity licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+accept "zilla://streams/net0"
+  option zilla:window 8192
+  option zilla:transmission "duplex"
+  option zilla:byteorder "network"
+
+accepted
+connected
+
+read  [0x10 0x13]             # CONNECT
+      [0x00 0x04] "MQTT"      # protocol name
+      [0x05]                  # protocol version
+      [0x02]                  # flags = clean start
+      [0x00 0x3c]             # keep alive = 60s
+      [0x00]                  # properties = none
+      [0x00 0x06] "client"    # client id
+
+write [0x20 0x08]             # CONNACK
+      [0x00]                  # flags = none
+      [0x00]                  # reason code
+      [0x05]                  # properties
+      [0x27] 66560            # maximum packet size = 66560
+
+read  [0xe0 0x02]             # DISCONNECT
+      [0x00]                  # normal disconnect
+      [0x00]                  # properties = none
+
+read closed
```

Comment:
Actually, this test is not needed anymore (I think). It supposed to test that if the client sends the session expiry in the CONNECT as 0, but sends non-zero in the disconnect it's a protocol error. Since we'll always override the 0 expiry in the CONNACK, I'm not sure if this test has any value. If we'd like to cover this scenario, we'd need to remember the original expiry, not just the "final" one.

Reply:
Removed this test so far, let me know what you think.

Reply:
TODO: Put back the test.